composer.json

```diff
@@ -26,7 +26,7 @@
         "illuminate/encryption": "^9.0|^10.0",
         "illuminate/http": "^9.0|^10.0",
         "illuminate/support": "^9.0|^10.0",
-        "lcobucci/jwt": "^3.4|^4.0",
+        "lcobucci/jwt": "^5.0",
```

Comment:
We can't drop the old versions as Passport still supports PHP 8.0

Reply:
I know. this is a temporary commit to force the CI to install v5, so I could see the conflicts.